WcaOnRails/app/views/competitions_mailer/notify_wcat_of_confirmed_competition.html.erb

```diff
@@ -1,83 +1,92 @@
-Competition Announcement Team, <br>
+Competition Announcement Team, 
+<br>
+<br>
+<strong><%= @confirmer.name %> has confirmed <%= link_to @competition.name, competition_url(@competition) %>.</strong>
 <br>
-<strong><%= @confirmer.name %> has confirmed <%= link_to @competition.name, competition_url(@competition) %>.</strong><br>
 
 <span class="<%= 'alert' if (@competition.start_date - Time.now.to_date).to_i <= 30 %>">
   The competition will take place on <%= wca_date_range(@competition.start_date, @competition.end_date) %> in <%= (@competition.start_date - Time.now.to_date).to_i %> days.
   <% if (@competition.start_date - Time.now.to_date).to_i <= 30 %>
-    There are less than 48 hours remaining for the competition to be announced. @Delegates, please respond to any WCAT concerns as soon as possible to ensure that the competition becomes announced. If the competition becomes less than 28 days away, the WCAT cannot announce it.
-    <% end %>
+    There are less than 48 hours remaining for the competition to be announced. @<%= "Delegate".pluralize(@competition.delegates.count) %>: please respond to any WCAT concerns as soon as possible to ensure that the competition becomes announced. If the competition becomes less than 28 days away, the WCAT cannot announce it.
+  <% end %>
   <br>
   <br>
 </span>
 
 <% if @competition.championships.present? %>
   <span class="alert">
-    This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %>.<br>
+    This competition is marked as <%= @competition.championships.sort.map(&:name).to_sentence %>.
+    <br>
     <br>
   </span>
-  <% end %>
+<% end %>
 
 <% @competition.delegates.map do |delegate| %> 
   <% if Team.probation.current_members.map { |member| member.user_id }.include?(delegate.id) %>
     <span class="alert">
-      <%= delegate.name %> is currently on probation. Please check the <%= link_to "Delegates on Probation spreadsheet", "https://docs.google.com/spreadsheets/d/1ZU1lk0niGn-vjmbiDkoROU8qjmdHkNebZ2GCFHm6vvA/edit#gid=1887522006" %> for more details.<br>
+      <%= delegate.name %> is currently on probation. Please check the <%= link_to "Delegates on Probation spreadsheet", "https://docs.google.com/spreadsheets/d/1ZU1lk0niGn-vjmbiDkoROU8qjmdHkNebZ2GCFHm6vvA/edit#gid=1887522006" %> for more details.
+      <br>
       <br>
     </span>
-    <% end %>
   <% end %>
+<% end %>
 
 <% @competition.nearby_competitions(Competition::NEARBY_DAYS_WARNING, Competition::NEARBY_DISTANCE_KM_WARNING).each do |c| %>
+  <% days_difference = (@competition.start_date - c.start_date).to_i %>
```

Comment:
This line (and the one below) should go into the `if` statement, as there's no need for it if it's not going to be used.